### ISSUE TITLE
[Bug]: ImportError: cannot import name 'model_urls' from 'torchvision.models.vgg' (F:\3\envs\Mt\lib\site-packages\torchvision\models\vgg.py)

### ISSUE THREAD

**Issue by My12123** [zyddnys/manga-image-translator#326]
### Issue

ImportError

### Command Line Arguments

```Shell
python -m manga_translator -v --mode batch --use-cuda --translator=papago -l RUS --font-path=fonts/anime_ace.ttf -i I:\Sammy
```

### Console logs

```Shell
(Mt) F:\manga-image-translator>python -m manga_translator -v --mode batch --use-cuda --translator=papago -l RUS --font-path=fonts/anime_ace.ttf -i I:\Sammy
Traceback (most recent call last):
  File "F:\3\envs\Mt\lib\runpy.py", line 187, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "F:\3\envs\Mt\lib\runpy.py", line 146, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "F:\3\envs\Mt\lib\runpy.py", line 110, in _get_module_details
    __import__(pkg_name)
  File "F:\manga-image-translator\manga_translator\__init__.py", line 7, in <module>
    from .manga_translator import *
  File "F:\manga-image-translator\manga_translator\manga_translator.py", line 18, in <module>
    from .args import DEFAULT_ARGS
  File "F:\manga-image-translator\manga_translator\args.py", line 4, in <module>
    from .detection import DETECTORS
  File "F:\manga-image-translator\manga_translator\detection\__init__.py", line 6, in <module>
    from .craft import CRAFTDetector
  File "F:\manga-image-translator\manga_translator\detection\craft.py", line 26, in <module>
    from .craft_utils.vgg16_bn import vgg16_bn, init_weights
  File "F:\manga-image-translator\manga_translator\detection\craft_utils\vgg16_bn.py", line 7, in <module>
    from torchvision.models.vgg import model_urls
ImportError: cannot import name 'model_urls' from 'torchvision.models.vgg' (F:\3\envs\Mt\lib\site-packages\torchvision\models\vgg.py)
```

**Comment by JustFrederik** [comment]
https://stackoverflow.com/questions/67317418/importerror-cannot-import-name-model-urls-from-torchvision-models-vgg

**Comment by My12123** [comment]
This appeared today when I updated manga-image-translator. How to fix it?

**Comment by JustFrederik** [comment]
You torchvision is too new. You could downgrade or just comment it out when you dont use the detector or you replace it like it is explained on stackoverflow

**Comment by My12123** [comment]
Is it possible to fix without changing the version? It worked before the update, but now it doesn't.

**Comment by My12123** [comment]
Can this be fixed or does it require a return to 376b21d ?
